x-pack/legacy/plugins/fleet/server/repositories/enrollment_api_keys/__memorize_snapshots__/default.contract.test.slap_snap

```diff
@@ -1031,3 +1031,341 @@ exports['Enrollment api key Repository delete allow to delete a enrollmentApiKey
     "saved_objects": []
   }
 }
+
+exports['Enrollment api key Repository create allow to create an enrollment api key - create:enrollment_api_keys (1)'] = {
```

Comment:
Not sure how this snapshot changes are related to the other changes in this PR. But CI is happy.

Reply:
It should not be related at all, we probably have some flakiness in our tests suite